identity/webapp/pages/api/users/[[...users]].ts

```diff
@@ -29,7 +29,16 @@ const handleIdentityApiRequest: NextApiHandler = auth0.withApiAuthRequired(
           },
           validateStatus: (status: number) => status >= 200 && status < 500,
         })
-        .catch(error => error.response);
+        .catch(error => {
+          if (error.response) {
+            return error.response;
+          }
+          console.error('Connection-level error (no response received)', error);
```

Comment:
```suggestion
          // This can occur if, e.g. the TLS certificate for the API has expired.
          console.error('Connection-level error (no response received)', error);
```